tests/observability/metrics/utils.py

```
@@ -777,7 +774,7 @@ def get_pvc_size_bytes(vm: VirtualMachineForTests) -> str:
 
 
 def validate_metric_value_greater_than_initial_value(
-    prometheus: Prometheus, metric_name: str, initial_value: float, timeout: int = TIMEOUT_4MIN
+    prometheus: Prometheus, metric_name: str, initial_value: float, timeout: int = TIMEOUT_4MIN, equals: bool = False
```

Ruff (0.14.10)

Unused function argument: `equals`

(ARG001)